### DISCUSSION TITLE
Documentation for the config.yaml file

### DISCUSSION THREAD

**Discussion by brentrobbins** [nhost/nhost#100] [Q&A] [answered]
Is there a place (If not would you mind adding it) where I can find the documentation covering the `config.yaml` file? I would like to know what are all of the possible values, what all they mean, how are they used, etc. Thank you.

**Comment by elitan** [comment, marked as answer]
There is no such place yet.

Most values correspond to environment variables for auth specified here: https://github.com/nhost/hasura-auth/blob/main/src/utils/env.ts

We'll add more details to this in our documentation in the future.

**Reply by brentrobbins** [reply]
OK, thanks for letting me know @elitan.
